common/logger/logger.go

```diff
@@ -15,25 +15,45 @@
 package logger
 
 import (
+	"errors"
 	"io"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
 
 	lcf "github.com/Robpol86/logrus-custom-formatter"
 	"github.com/alecthomas/kingpin"
 	log "github.com/sirupsen/logrus"
 )
 
-// GetLogFormatter returns a configured logrus.Formatter for logging
-func GetLogFormatter(progName string) log.Formatter {
-	template := "%[ascTime]s %-5[process]d " + progName + "  %-7[levelName]s %[message]s %[fields]s\n"
-	return lcf.NewFormatter(template, nil)
-}
-
 // enableVerboseLogging enables verbose logging
 func enableVerboseLogging(ctx *kingpin.ParseContext) error {
 	log.SetLevel(log.DebugLevel)
 	return nil
 }
 
+// getCallerInfo returns the file and file line-number of a caller
+func getLogCallerInfo(e *log.Entry, f *lcf.CustomFormatter) (interface{}, error) {
+	var skip int = 1
```

Comment:
should omit type int from declaration of var skip; it will be inferred from the right-hand side